src/ai/backend/common/validators.py

```diff
@@ -479,6 +481,8 @@ def __init__(self, *, min_length: Optional[int] = None, max_length: Optional[int
             raise TypeError('min_length must be less than or equal to max_length when both set.')
         self._min_length = min_length
         self._max_length = max_length
+        if self._ascii_only:
+            type(self)._rx_slug = t.RegexpRaw(regexp=type(self)._regexp, re_flags=re.ASCII)
```

Comment:
🚨 Assigning to the class variable will change the behavior of all other trafaret instances!
Just keep both ASCII and Unicode versions of regex trafarets as the class variable and branch when checking.

Reply:
Or, use an instance variable and assign different versions of trafaret to avoid branching at runtime.

Reply:
Oops, my mistakes. I'll fix it right away!